.github/workflows/template-tests.yml

```
@@ -0,0 +1,40 @@
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  test-starter-projects:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)